### PR TITLE
Setuptools conversion

### PR DESCRIPTION
**Pull request by jhunkeler**
Help wanted.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,50 +1,5 @@
...
-[install_data]
-pre-hook.glob-data-files = stsci.distutils.hooks.glob_data_files
-
-[build_ext]
-pre-hook.pyraf-buildext-hook = pyraf_setup.build_ext_hook
+[bdist_wheel]
+# This flag says that the code is written to work on both Python 2 and Python
+# 3. If at all possible, it is good practice to do this. If you cannot, you
+# will need to generate wheels for each Python version that you support.
+universal=1
```
Comment (cdsontag) [inline setup.cfg]: This is not correct exactly.  The code is written to work on Py2, and to work on Py3 after 2to3 is run (with no hand editing).  Should we still have ```universal=1``` set?  I'm thinking not, but you know more about it.

File: setup.py [hunk 2]
```diff
@@ -1,15 +1,73 @@
...
+if os.path.exists('relic'):
+    sys.path.insert(1, 'relic')
+    import relic.release
+else:
+    try:
+        import relic.release
+    except ImportError:
+        try:
+            subprocess.check_call(['git', 'clone',
+                'https://github.com/jhunkeler/relic.git'])
```
Comment (cdsontag) [inline setup.py]: Let's not release code that goes to your personal repo.  Isn't there an SSB location to put relic?

File: setup.py [hunk 3]
```diff
@@ -1,15 +1,73 @@
...
+    classifiers = [
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Topic :: Scientific/Engineering :: Astronomy',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+    ],
+    install_requires = [
+        'astropy',
```
Comment (cdsontag) [inline setup.py]: does not require astropy or nose or scipy or sphinx.  just numpy.  let's not bloat this to bigger than it has to be.

**Comment by jhunkeler** [comment]
Sorry Chris, I should have mentioned I wasn't done porting this package yet (it's a copy/paste/sed-replaced boilerplate at the moment). There are a lot of little details to flesh out with this one.

**Comment by pllim** [comment]
If you still need this, need to rebase to resolve conflict. FYI.

**Pull request closed without merging**